messenger.rst

```diff
@@ -1980,6 +1980,11 @@ A handler class can handle multiple messages or configure itself by implementing
         }
     }
 
+.. deprecated:: 6.2
+
+    :class:`Symfony\\Component\\Messenger\\Handler\\MessageSubscriberInterface` was deprecated in Symfony 6.2.
+    Use `#[AsMessageHandler]` attribute instead.
```

Comment:
```suggestion
    Use :class:`..... <#[AsMessageHandler]>` attribute instead.
```

Reply:
I don't know this syntax, what is it for? Why not put `:class:Symfony\\Component\\Messenger\\Attribute\\AsMessageHandler` ?

Reply:
Oskar actually showed the example abbreviated and the wrong way around :)

It should be: ``` :class:`#[AsMessageHandler] <Symfony\\Component\\Messenger\\Attribute\\AsMessageHandler>` ```

In this case, the link will be rendered as [`#[AsMessageHandler]`](https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/Messenger/Attribute/AsMessageHandler.php). The part in between the `<...>` brackets is the "link target" and the part before it is the rendered label.

Reply:
Thanks Wouter, sorry I mixed things up.

Reply:
Thanks for the explanation Wouter, and thanks for the review Oskar.